wg-security-audit/RFP.md

```diff
@@ -96,4 +96,11 @@ The audit should result in the following artifacts, which will be made public af
 
 ## Q & A
 
-This section intentionally left empty until the question period opens.
+| # | Question | Answer |
+|---|----------|--------|
+| 1 | The RFP says that any area included in the out of scope section of the k8s bug bounty programme is not in-scope of this review.  There are some areas which are out of scope of the bug bounty which would appear to be relatively core to k8s, for example Kubernetes on Windows. Can we have 100% confirmation that these areas are out of scope? | Yes. If you encounter a vulnerability in Kubernetes' use of an out-of-scope element, like etcd or the container network interface (to Calico, Weave, Flannel, ...), that is in scope. If you encounter a direct vulnerability in a third-party component during the audit, you can report it to them.  |
```

Comment:
Perhaps we should be the ones to handle comms with third parties.

Reply:
Yeah I think we say this better in question 3. Maybe copy that language into this?

'If you encounter a direct vulnerability in a third-party component during the audit you should follow the embargo section of the RFP.'

Reply:
perfect